### DISCUSSION TITLE
[Bug report] Synchronization failure

### DISCUSSION THREAD

**Discussion by ProyectoFlower** [GetPublii/Publii#1800] [Bug Report]
### Operating system

Windows 10 Pro

### Publii version

0.46.2 (build 16966)

### Issue type

Application

### Bug description

I am experiencing a very rare bug, I finish editing my website and sync it to Github. Everything normal, the progress bar is slowly filling up as usual.

![1](https://github.com/user-attachments/assets/547c9bd9-3e47-47a8-8498-f5a29a0ed1c8)

But suddenly I get this error message (or similar) when it finishes uploading all the files.

![2](https://github.com/user-attachments/assets/6a09616f-2c1f-43dd-9f82-ab69cc1fcf24)

And to make it worse, when trying to sync the website several times, it finally manages to sync with Github. Basically, it fails between 5 to 6 times and on the 7th time it succeeds.

And in case you're wondering, I have my Github Pages connection settings configured properly.

### Steps to reproduce

Just press the button to sync the site.

**Comment by dziudek** [comment]
@ProyectoFlower - looks like some connectivity or API issue. We will make improvements for Github sync in v.0.47 so maybe then also this error will disappear. I will note to inform you when changes will be available
